src/modules/previewpane/MonacoPreviewHandler/generateLanguagesJson.html

```diff
@@ -18,7 +20,8 @@
                 document.body.removeChild(element);
             }
             require.config({ paths: { vs: 'monacoSRC/min/vs' } });
-            require(['vs/editor/editor.main'], function () {
+            require(['vs/editor/editor.main'], async function () {
+		        await registerAdditionalLanguages(monaco);
```

Comment:
indentation

Reply:
In this document everything seems normal to me